src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs

```diff
@@ -9,6 +9,353 @@ namespace System.Text.Json
 {
     public sealed partial class Utf8JsonWriter
     {
+        /// <summary>
+        /// Writes the pre-encoded property name (as a JSON string) as the first part of a name/value pair of a JSON object.
+        /// <param name="propertyName">The JSON encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
+        /// </summary>
```

Comment:
param doc is within the summary tag, flip this line and the one above.

Reply:
Oops. Good catch.